pubspec.yaml

```diff
@@ -15,8 +15,8 @@ dependencies:
 
   enough_html_editor:
     git:
-      url: https://github.com/linagora/enough_html_editor.git
-      ref: email_supported
+      url: https://github.com/aryan-more/enough_html_editor.git
+      ref: "patch-1"
```

Comment:
@aryan-more We have merged the PR https://github.com/linagora/enough_html_editor/pull/24 into the `upgrade_v3.16.0` branch. Please use it to test on version `3.16.0`

Reply:
Hey @dab246 i have updated git repo in dependency to use upgrade_v3.16.0 branch